kubernetesevents/namespace_handler.go

```diff
@@ -0,0 +1,59 @@
+package kubernetesevents
```

Comment:
Whats the fundamental difference between this and GenericHandler.  Why didn't GenericHandler suffice?